src/main/java/core/basesyntax/service/TransactionService.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.FactoryStrategy;
+import core.basesyntax.strategy.OperationService;
+import java.util.List;
+
+public class TransactionService {
+    private final FactoryStrategy factoryStrategy;
+
+    public TransactionService() {
+        factoryStrategy = new FactoryStrategy();
```

Comment:
let's pass the factoryStrategy as an argument to the TransactionService constructor